ci/provisioning/post_provision_config_nodes_LEAP_15.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+post_provision_config_nodes() {
+    # TODO: port this to Zypper
+    #       or do we even need it any more?
+    #if $CONFIG_POWER_ONLY; then
+    #    rm -f /etc/yum.repos.d/*.hpdd.intel.com_job_daos-stack_job_*_job_*.repo
+    #    yum -y erase fio fuse ior-hpc mpich-autoload               \
+    #                 ompi argobots cart daos daos-client dpdk      \
+    #                 fuse-libs libisa-l libpmemobj mercury mpich   \
+    #                 openpa pmix protobuf-c spdk libfabric libpmem \
+    #                 libpmemblk munge-libs munge slurm             \
+    #                 slurm-example-configs slurmctld slurm-slurmmd
+    #fi
+    
+    source /etc/os-release
+    # Temp fix for broken mirror until snapshot is rebuilt to not use it.
+
+    if [ $VERSION_ID == '1.5' ]; then
```

Comment:
(lint) Couldn't find 'fi' for this 'if'. [SC1046]
(lint) Couldn't parse this if expression. [SC1073]